src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -99,7 +99,9 @@ def assertRegConditionalsForPpaRefModel(registration_requests,
           # If the cbsdCategory is not present in registration request then
           # assign it to the cbsdCategory in conditional_params.
           if 'cbsdCategory' not in device:
-            break
+            device.update({'cbsdCategory': conditional_params.get('cbsdCategory',
+                                                                  'B')})
```

Comment:
There is no default CBSD category, so you can't use `get`  here.

Reply:
FW:
Would this be an acceptable fix, to NOT use a default category, since cbsdCategory seems to be required for conditionalRegistrationData?

Kate:
I think that `get` still fills in `None` in that case (rather than raising an exception) if the field isn't in conditional_params. It seems better to raise an error -- indicating a bad config -- than to silently ignore the problem and let the error be caught later (which I think it would be?).